src/sync.rs

```diff
@@ -268,6 +268,17 @@ impl Context {
             .log_err(self)
             .ok();
         }
+
+        // Since there was a sync message, we know that there is a second device.
+        // Set BccSelf to true if it isn't already.
+        if !items.items.is_empty() && !self.get_config_bool(Config::BccSelf).await.unwrap_or(false)
```

Comment:
`!= Ok(true)` is easier to read, but i'd make it `== Ok(false)`, i think we don't want to set the config on failures.

Reply:
PartialEq isn't implemented for anyhow::Result, so this doesn't work unfortunately. But I changed the code not to set the config on errors.